### PR TITLE
doc: 📝 Update doc to use OVHcloud image

### PR DESCRIPTION
**Pull request by philippart-s**
PR to update the autoscaler demo with official OVHcloud Docker image

### CODE REVIEW

File: pages/public_cloud/containers_orchestration/managed_kubernetes/cluster-autoscaler-example/guide.en-gb.md [hunk 1]
```diff
@@ -29,305 +29,241 @@ You can activate the autoscaler on several node pools, each of which can have a
...
 ```
 
 As you can see then with `kubectl get nodepools`, the autoscaler detects capacity has been reached and asks for a new node:
 
 ```console
 $ kubectl get nodepools
-NAME            FLAVOR   AUTO SCALED   [...]   ANTI AFFINITY   DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   MIN   MAX
-nodepool-b2-7   b2-7     true          [...]   false           2         1         1            1           0     3
+
+kubectl get nodepools
```
Comment (scraly) [inline pages/public_cloud/containers_orchestration/managed_kubernetes/cluster-autoscaler-example/guide.en-gb.md]: the command is displayed two times? :)

Comment (philippart-s) [reply pages/public_cloud/containers_orchestration/managed_kubernetes/cluster-autoscaler-example/guide.en-gb.md]: fix https://github.com/ovh/docs/pull/5940/commits/bcc1b4a09339b41ea5fb0aadddc70dc9d6d9c97d

File: pages/public_cloud/containers_orchestration/managed_kubernetes/cluster-autoscaler-example/guide.en-gb.md [hunk 2]
```diff
@@ -29,305 +29,240 @@ You can activate the autoscaler on several node pools, each of which can have a
...
 
 ## Deploying a test workload
 
-In order to test the autoscaler, we propose you to install a *Prime Numbers* Deployment that deploys several instances of *Prime Numbers* pods. The *Prime Numbers* pod simply calculates prime numbers in the most performance ineffective way: by dividing every positive integer number by all the lower positive integers. It's a CPU intensive operation, but it uses a minimal amount of memory.  
+Let's assume that you have created an MKS cluster with a node pool with its minimum set to 1 and its maximum set to 3.
 
-> [!primary]
->
-> The *Prime Numbers* pod is a small [NodeJS](https://nodejs.org/) programs. The main script, on `index.js`, is a [Express server](https://expressjs.com/) that launches a [worker thread](https://nodejs.org/api/worker_threads.html) were the prime number calculation is done outside the event loop.
+In order to test the autoscaler, we offer you to install a *Python heavy CPU load* Deployment that deploys several instances of *Python CPU load* pods. The *Python CPU load* pod simply heavy charge each allocated CPU of your pod. It's a CPU intensive operation, but it uses a minimal amount of memory.
```
Comment (OVH-Team-Guides) [inline pages/public_cloud/containers_orchestration/managed_kubernetes/cluster-autoscaler-example/guide.en-gb.md]: "The *Python CPU load* pod simply heavy charge each allocated CPU of your pod."
Isn't a verb missing here ? I don't understand that sentence.

Comment (philippart-s) [reply pages/public_cloud/containers_orchestration/managed_kubernetes/cluster-autoscaler-example/guide.en-gb.md]: yes agree :), is it better: "The *Python CPU Load* pod aims to consume all the CPU allocated to it" ?

Comment (OVH-Team-Guides) [reply pages/public_cloud/containers_orchestration/managed_kubernetes/cluster-autoscaler-example/guide.en-gb.md]:
```suggestion
In order to test the autoscaler, we offer you to install a *Python heavy CPU load* Deployment that deploys several instances of *Python CPU load* pods. The *Python CPU load* pod's goal is to consume all the CPU allocated to it. It's a CPU intensive operation but it uses a minimal amount of memory.
```

**Comment by OVH-Team-Guides** [comment]
Thank you for your contribution @philippart-s

**Pull request merged**